### PR TITLE
fix(generators/react-cmf): updated packages

### PR DESCRIPTION
**Pull request by vyshkov**
**What is the problem this PR is trying to solve?**

* if you generate react-cmf application with the latest version of talend-generator - it will not run. Because all imports now have `@talend` scope, but `package.json` still has old talend packages.
* webpack version outdated
* other errors

**What is the chosen solution to this problem?**
* updated @talend packages in `package.json`
* added `redux-saga`, `react-ace` (because of errors)
* updated webpack version (to 3) and configuration
* changed loaders configuration

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/generator/generators/react-cmf/templates/package.json [hunk 1]
```diff
@@ -41,19 +41,21 @@
...
+    "resolve-url-loader": "^2.1.0",
+    "sass-loader": "6.0.5",
+    "style-loader": "0.18.2",
     "url-loader": "0.5.8",
-    "webpack": "^1.14.0",
-    "webpack-dashboard": "0.4.0",
-    "webpack-dev-server": "^1.16.5"
+    "webpack": "^3.6.0",
+    "webpack-dashboard": "^1.0.0-5",
+    "webpack-dev-server": "^2.5.0"
```
Comment (jmfrancois) [inline packages/generator/generators/react-cmf/templates/package.json]: webpack 3 is not supported accros the stack at the moment (not everywhere)
please align using version script provided in UI.
I think we should make the script update this package.json along the others

File: packages/generator/generators/react-cmf/templates/webpack.config.js [hunk 2]
```diff
@@ -1,79 +1,68 @@
...
+
+const extractCSS = new ExtractTextPlugin({ filename: '[name]-[hash].css' });
 
 const SASS_DATA = `$brand-primary: #77828A;
-@import '~bootstrap-talend-theme/src/theme/guidelines';
+@import '~@talend/bootstrap-theme/src/theme/guidelines';
 `;
 
+function getCommonStyleLoaders(enableModules) {
+	const cssOptions = enableModules ?
```
Comment (talend-glorieux) [inline packages/generator/generators/react-cmf/templates/webpack.config.js]: Please don't use multi-line ternaries.

File: packages/generator/generators/react-cmf/templates/webpack.config.js [hunk 3]
```diff
@@ -1,79 +1,68 @@
...
-	},
-	sassLoader: {
-		includePaths: [
-			path.resolve(
-				__dirname,
-				'node_modules'
-			),
+				test: /\.woff(2)?(\?v=\d+\.\d+\.\d+)?$/,
+				loader: 'url-loader',
+				options: { name: './fonts/[name].[ext]', limit: 10000, mimetype: 'application/font-woff' },
```
Comment (talend-glorieux) [inline packages/generator/generators/react-cmf/templates/webpack.config.js]: Where is t`10000` coming from?

Comment (vyshkov) [reply packages/generator/generators/react-cmf/templates/webpack.config.js]: Previously it was 50000 :) What value we should take ?

File: version.js [hunk 4]
```diff
@@ -2,9 +2,25 @@
 
 const fs = require('fs');
 const path = require('path');
-const cpx = require('cpx');
 const program = require('commander');
 
+const stack_version = program.stack || require('./lerna.json').version;
```
Comment (jmfrancois) [inline version.js]: program.stack is available before the parse ?

**Pull request merged**